packages/@react-spectrum/tree/docs/TreeView.mdx

```diff
@@ -26,7 +26,8 @@ import Image from '@spectrum-icons/workflow/Image';
 import Edit from '@spectrum-icons/workflow/Edit';
 import Delete from '@spectrum-icons/workflow/Delete';
 import {Text} from '@react-spectrum/text';
-import {TreeView, TreeViewItem} from '@react-spectrum/tree';
+import {TreeView, TreeViewItem, TreeItemContent} from '@react-spectrum/tree';
+import {Collection} from 'react-aria-components';
```

Comment:
Should we re-export this from `@react-spectrum/tree` (and eventually the RSP monopackage)?